entry_types/scrolled/package/src/frontend/Backdrop.js

```diff
@@ -129,6 +130,21 @@ function BackgroundVideo(props) {
       playerActions.pause()
     }
   });
+  
+  useEffect(() => {
+    let callback = (visibilityState) => {
+      if (visibilityState === 'hidden') {
+        playerActions.pause();
+      }
+      else{
+        playerActions.play();
+      }
+    }
+    documentHiddenState(callback); //register callback
+    return () => {
+      documentHiddenState(callback, true); //remove callback
+    }
```

Comment:
If `documentHiddenState` returned a function that removes the event listeners, it would be a better fit for `useEffect`. We would also not need the comments, that currently work around the fact that the `true` parameter does not convey meaning.

When a background video is playing and atmo played at least once before, switching away from the tab resumes the atmo. This  happens since all media players attach atmo hooks - even those for background videos. Switching the tab now pauses the video and calls the after hook which resumes the atmo.

Reply:
so background videos should not have atmo hooks right?

Reply:
In the recent changes I have adjusted the documentHiddenState as you suggested.
Now only assigning hooks to InlineVideo and InlineAudio elements.
BackgroundVideos player should not be getting hooks assigned.
Since Inline players are not getting paused on visibilitychange, this problem should be resolved now?

Reply:
Should we stop Atmo when BackgroundVideo starts playing?
I realised it today that BackgroundVideo has sound as well

Reply:
I would do it the same way we currently do it in paged entries, because there's no reason to do anything different here